docs/quantum_research/annealing.rst

```diff
@@ -680,7 +680,7 @@ anneal in reverse from there. This section describes these features.
 Pause and Quench
 ----------------
 
-|dwave_5kq| systems provide user control over the global annealing trajectories:
+QPU solvers provide user control over the global annealing trajectories:
```

Comment:
```suggestion
Quantum computers provide user control over the global annealing trajectories:
```

This page has "solver" in page references but not otherwise (solver is higher up in the software stack, this page is the lowest physical level)